Formula/manifest-tool.rb

```diff
@@ -0,0 +1,33 @@
+class ManifestTool < Formula
+  desc "Command-line tool to create and query container image manifest list/indexes"
+  homepage "https://github.com/estesp/manifest-tool/"
+  url "https://github.com/estesp/manifest-tool/archive/refs/tags/v2.0.3.tar.gz"
+  sha256 "53bba7cc7465c8347c70e36467848bd385eba2fb51d7aceb269c316382a065a7"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    system "make", "install", "INSTALLDIR=#{bin}"
```

Comment:
```suggestion
    system "make", "install", "PREFIX=#{prefix}"
```
This works better if they start installing things outside of `bin`. It seems like they might, since there's a reference to a man directory in the `Makefile`.